packages/core/src/indexing/service.ts

```diff
@@ -602,8 +601,6 @@ const executeLog = async (
 
     const decodedCheckpoint = decodeCheckpoint(event.encodedCheckpoint);
 
-    error.meta = `Event arguments:\n${prettyPrint(event.event.args)}`;
```

Comment:
I removed them because we already have the same logic in the indexing-store,  which adds a more useful error message because it has different context than the indexing service. I think this line was actually a bug, because setting `error.meta` was immediately overwriting the message from the indexing-store.

I manually verified that the messages were what we wanted for both the case when an error is throw and when an error is thrown that includes an extra message (like `replaceBigInt` or "forgot to use await")